app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -138,120 +124,45 @@ function ReviewIssuesStep({
 
             {remainingAttempts <= DISPLAY_ATTEMPTS && (
               <p>
-                <strong>{t('idv.failure.attempts', { count: remainingAttempts })}</strong>
+                <strong>{t('idv.failure.attempts_html', { count: remainingAttempts })}</strong>
               </p>
             )}
           </Warning>
           <Cancel />
         </>
       );
     }
-    if (inPersonCtaVariantActive === 'in_person_variant_a') {
-      return (
-        <Warning
-          heading={t('errors.doc_auth.throttled_heading')}
-          actionText={t('idv.failure.button.warning_variant')}
-          actionOnClick={onWarningPageDismissed}
-          location="doc_auth_review_issues"
-          remainingAttempts={remainingAttempts}
-          troubleshootingOptions={
-            <DocumentCaptureTroubleshootingOptions
-              location="post_submission_warning"
-              showAlternativeProofingOptions={!isFailedResult}
-              heading={t('components.troubleshooting_options.ipp_heading')}
-              altInPersonCta={t('in_person_proofing.headings.cta_variant')}
-              altInPersonCtaButtonText={t('in_person_proofing.body.cta.button_variant')}
```

Comment:
There are some lingering references to these props that I think can be removed now?

https://github.com/18F/identity-idp/blob/d7de89c965bd9c3ceed1a684e43050ad81133e0b/app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.tsx#L37-L39

https://github.com/18F/identity-idp/blob/d7de89c965bd9c3ceed1a684e43050ad81133e0b/app/javascript/packages/document-capture/components/in-person-call-to-action.tsx#L8-L10

Reply:
yes that makes sense. I just applied a patch Tomas created to remove this

Reply:
done in 66f1e900cdc71c20c7edc348f29510845e7ab9d4